PDSA_System/Server/Controllers/BrukerController.cs

```diff
@@ -83,11 +84,10 @@ public async Task<ActionResult<List<Bruker>>> CreateBruker(Bruker bruker)
 
             return Ok(await GetBruker(bruker.AnsattNr));
         }
-
         /*
          Updater en Bruker --> ikke helt funksjonell enda.
          */
-        [HttpPut("/api/[Controller]/admin/UpdateBruker")]
+        [HttpPut("/api[Controller]/admin/UpdateBruker/{AnsattNr}")]
```

Comment:
Shit, glemte dette, mangler en skråstrek
```suggestion
        [HttpPut("/api/[Controller]/admin/UpdateBruker/{AnsattNr}")]
```